src/main/java/org/isf/stat/manager/JasperReportsManager.java

```diff
@@ -1097,4 +1099,37 @@ private static Date toDate(LocalDate localDate) {
 		return Optional.ofNullable(localDate).map(t -> Date.from(t.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant())).orElse(null);
 	}
 
-}
+	public JasperReportResultDto getInventoryReportPdf(MedicalInventory inventory, String jasperFileName, int printQtyReal) throws OHServiceException {
+		try {
+			HashMap<String, Object> parameters = new HashMap<>();
```

Comment:
```suggestion
			Map<String, Object> parameters = new HashMap<>();
```
The definition of the variable should use the interface not the concreate class.

Reply:
Thank you.